library_samples/Python/setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="ocs_sample_library_preview",
-    version="0.1.10_preview",
```

Comment:
can we update the requests package to 2.24 here?

Reply:
Updated